doc/info.md

```diff
@@ -61,19 +61,19 @@ Data bases:
 
 ## Operator selector.
 
-We divide the blocks of the blockchain in slots and eras.  An era has 20 slots and a slot has 100 blocks.
+Each block is an etherum block which is every 15 seconds. We then have an era which is 20000 blocks and a slot has 100 blocks. That means that an Era lasts 5.5 hours and a slot is 25 minutes. 
```

Comment:
- `era` which is 20 slots
- `slot` which is 100 block ethereum
- `era` lasts 8.3 hours

Reply:
Ah cool. Let me fix this in the morning :)